src/drift/models/base.py

```diff
@@ -7,12 +7,6 @@
 
 
 class Model(Transformation):
-    @abstractmethod
-    def fit(
-        self, X: pd.DataFrame, y: pd.Series, sample_weights: Optional[pd.Series] = None
-    ) -> None:
-        raise NotImplementedError
-
```

Comment:
Why did you remove this? Isn't this a basic api of Models anymore?

Reply:
the exactly same method is present on `Transformation`, which `Model` subclasses, so this is duplicate definition!